python/test/unit/language/test_core.py

```diff
@@ -1745,6 +1745,38 @@ def kernel_r(ptrs, BLOCK_SIZE: tl.constexpr):
     out = kernel_r[(2, )](data, BLOCK_SIZE=block_size, num_ctas=1, launch_cooperative_grid=False)
 
 
+@pytest.mark.interpreter
+@pytest.mark.skipif(is_hip(), reason="Not implemented for AMD At this moment")
+def test_load_scope_sem(device):
```

Comment:
I wonder if we still need this test at all. The lit test is probably enough?

Reply:
I only did it to do an end to end test, but I could try and make a lit test that makes sure the right PTX is generated too.

BTW I had also added a global variable to disable so that it is disable-able as you and Phil asked for the previous version.